app/lib/experience.js

```diff
@@ -8,12 +8,12 @@ module.exports = Base => class extends Base {
         return new Promise((resolve) => {
             const listenForData = (data) => {
                 let text = data.toString();
-                let regEx = new RegExp(`.*\\s${playerName}\\shas\\s\\d+\\sexperience\\s${getData}`);
+                let regEx = /.*Joshyray has (\d+) experience.*/ig;
```

Comment:
Wouldn't hardcoding Joshyray only check for your exp specifically?

Reply:
fixed